administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -257,9 +255,8 @@
 </button>
 
 <form action="<?php echo JRoute::_(
-			'index.php?option=com_associations&view=association&layout=' . $layout . '&acomponent='
-			. $aComponent . '&aview=' . $aView . '&extension=' . $extension . '&referencelanguage=' . $rLanguage . '&id='
-			. $this->referenceId
+			'index.php?option=com_associations&view=association&layout=' . $layout . '&component='
+			. $component . $rLanguage . '&id=' . $this->referenceId
```

Comment:
this will not work ...

instead of
$component . $rLanguage
it should be
$component . '&referencelanguage=' . $rLanguage